### PR TITLE
Bump lifecycle library version

### PR DESCRIPTION
**Pull request by natalieparellano**

### CODE REVIEW

File: internal/build/lifecycle_execution.go [hunk 1]
```diff
@@ -860,11 +851,11 @@ func (l *LifecycleExecution) hasExtensionsForRun() bool {
 }
 
 func (l *LifecycleExecution) runImageAfterExtensions() string {
-	var amd analyzedMD
+	var amd files.Analyzed
 	if _, err := toml.DecodeFile(filepath.Join(l.tmpDir, "analyzed.toml"), &amd); err != nil {
 		return l.opts.RunImage
 	}
-	if amd.RunImage == nil {
+	if amd.RunImage == nil || amd.RunImage.Image == "" {
```
Comment (natalieparellano) [inline internal/build/lifecycle_execution.go:858]: It looks like `github.com/BurntSushi/toml` had a bug before where empty structs without an `omitempty` annotation weren't getting written out to TOML files, so in our fixture file `run-image` was not present and `amd.RunImage` actually was `nil`. This change fixes the test no matter which version of the library we use.

cc @jkutner

File: acceptance/acceptance_test.go [hunk 2]
```diff
@@ -2643,6 +2643,7 @@ include = [ "*.jar", "media/mountain.jpg", "/media/person.png", ]
 								"rebase", repoName,
 								"--run-image", runAfter,
 								"--pull-policy", "never",
+								"--force",
```
Comment (natalieparellano) [inline acceptance/acceptance_test.go:2646]: Worth noting, the need for `--force` is a breaking change. We should call this out in the release notes.

**Review by joe-kimmel-vmw — APPROVED** [review]
LGTM

File: internal/build/lifecycle_execution.go (hunk 1 above)
Comment (jkutner) [reply internal/build/lifecycle_execution.go:858]: nice! thanks

**Review by jkutner — APPROVED** [review]

**Pull request merged**